src/assets/Generator.Shared/Utf8JsonWriterExtensions.cs

```diff
@@ -140,5 +140,33 @@ public static void WriteObjectValue(this Utf8JsonWriter writer, object value)
                     throw new NotSupportedException("Not supported type " + value.GetType());
             }
         }
+
+        public static void WriteBinaryDataValue(this Utf8JsonWriter writer, BinaryData value)
+        {
+            string inputValue = value.ToString();
+            // Check if it is possible to serialize the value as json value.
+            if (inputValue.StartsWith("{", StringComparison.InvariantCultureIgnoreCase) && (inputValue.EndsWith("}", StringComparison.InvariantCultureIgnoreCase) || inputValue.EndsWith($"}}{Environment.NewLine}", StringComparison.InvariantCultureIgnoreCase)) ||
+                inputValue.StartsWith("[", StringComparison.InvariantCultureIgnoreCase) && (inputValue.EndsWith("]", StringComparison.InvariantCultureIgnoreCase) || inputValue.EndsWith($"]{Environment.NewLine}", StringComparison.InvariantCultureIgnoreCase)) ||
+                inputValue.Equals("true", StringComparison.Ordinal) || inputValue.Equals("false", StringComparison.Ordinal) ||
+                Char.IsDigit(inputValue.First()) && Char.IsDigit(inputValue.Last()))
+            {
+                try
+                {
+#if NET6_0_OR_GREATER
+				    writer.WriteRawValue(value);
+#else
+                    JsonSerializer.Serialize(writer, JsonDocument.Parse(inputValue).RootElement);
+#endif
+                }
+                catch
```

Comment:
Please specify the exception you're catching here.

Reply:
specified.